src/modules/AppComponents/displayMovies.js

```diff
@@ -0,0 +1,54 @@
+import mapShowDto from '../Helpers/createShowObj.js';
+import { AddLike, getLikes } from '../InvolvementService/LikeService.js';
+import { getShowById } from '../TvMazeService/TvMazeService.js';
+import displayLikes from './displayLikes.js';
+import renderCommentPopup from './displayPopUp.js';
+
+const generateGenres = (genresList) => genresList.reduce((List, genre) => `${List}<li>${genre}</li>`, '');
+
+const generateMovie = (movie) => `
+  <article id=${movie.id} class="movie-card">
+    <img src="${movie.image.medium}" class="card-img" alt="movie image" />
+    <div class="card-footer">
+      <h3 class="movie-title">${movie.name}</h3>
+      <ul class="movie-tags">
+        ${generateGenres(movie.genres)}
+      </ul>
+      <div class="action">
+        <button class="card-btn">See more</button>
+        <div class="likes-container">
+          <i class="fa-solid fa-heart"></i>
```

Comment:
[Optional] It would be better if you can disable adding more likes after the first click on the heart icon. And also changing it's background color after clicking.